src/protocol.rs

```diff
@@ -283,6 +316,7 @@ impl PayloadBuffer {
             num_messages: 0,
             data: Header {
                 magic: MAGIC,
+                brightness: 0,
```

Comment:
Just to make it easier to understand:
```suggestion
                brightness: Brightness::Full.into(),
```